chia/protocols/protocol_timing.py

```diff
@@ -0,0 +1,4 @@
+# These settings should not be end-user configurable
+INVALID_PROTOCOL_BAN_SECONDS = 10
```

Comment:
I stuck these in here because I didn't want users fiddling with them as config settings.